packages/ui/src/Table/index.tsx

```diff
@@ -82,23 +91,30 @@ Table.Thead = Thead;
 const Tbody = ({ children }: PropsWithChildren) => <tbody>{children}</tbody>;
 Table.Tbody = Tbody;
 
-const Tr = ({ children }: PropsWithChildren) => (
-  <tr className='[&>td:last-child]:border-b-0'>{children}</tr>
-);
+const Tr = ({ children }: PropsWithChildren) => <tr>{children}</tr>;
 Table.Tr = Tr;
 
 type HAlign = 'left' | 'center' | 'right';
 type VAlign = 'top' | 'middle' | 'bottom';
 
-const getCell = (density: Density) =>
-  cn('box-border', 'pl-3 pr-3', density === 'sparse' ? 'pt-4 pb-4' : 'pt-3 pb-3');
+const densityPaddingMapping = {
+  slim: 'px-3 py-2',
+  compact: 'px-3 py-3',
+  sparse: 'px-3 py-4',
+};
+
+const getCell = (density: Density) => {
+  const padding = densityPaddingMapping[density];
+  return cn('box-border', padding);
+};
```

Comment:
question: i know it might be too much but maybe it's worth to implement the `TableCell` unified component that is designed in Figma? We would have to do it one day anyway, so why not now? I believe it would suit all our needs for position table

https://www.figma.com/design/WKgXSVVK3L0EI0sr9Hwjoa/penumbra-UI-library?node-id=333-17181&m=dev

Reply:
That we probably don't need `Table`, `TableRow`, `TableHeading` anymore with the new `TableCell` component from Figma

Reply:
Not pushing for this to be in this PR. We can do it later and merge this pr

Reply:
Got it